source/sdk/math.ooc

```diff
@@ -68,14 +68,10 @@ extend Int64 {
 
 extend Int {
 	modulo: func (divisor: This) -> This {
-		if (divisor < 0) {
-			this *= -1
-			divisor *= -1
-		}
-		result := this % divisor
+		result := this
```

Comment:
It is not strictly necessary to introduce `result` as you could as well modify `this` to contain your result as this is a call by value member of a cover but this way is probably much clearer.